static/src/javascripts/boot.js

```diff
@@ -181,7 +187,7 @@ const go = () => {
             : Promise.resolve({ bootEnhanced: () => { } });
 
         Promise.all([
-            fetchCommercial.then(({ bootCommercial }) => {
+            fetchCommercial().then(({ bootCommercial }) => {
```

Comment:
Just acknowledging the noop pattern is a bit convoluted. It seems to only exist for commercial-legacy now. I'll add this to a clean up ticket.